lib/screens/auth/auth_screen.dart

```diff
@@ -0,0 +1,157 @@
+    );
+  }
+}
+
+class AuthScreenContent extends StatefulWidget {
+
+  @override
+  _AuthScreenContentState createState() => _AuthScreenContentState();
+}
+
+class _AuthScreenContentState extends State<AuthScreenContent> {
+  final GlobalKey<FormState> _key = new GlobalKey();
+  bool _validate = false;
+  final LoginRequestData _loginData = LoginRequestData();
+  bool _obscureText = true;
+
+  @override
+  Widget build(BuildContext context) {
+    LogInController _logInModel = Provider.of<LogInController>(context);
+
+    return new Scaffold(
+      body: new Center(
+        child: new SingleChildScrollView(
+          child: new Container(
+            margin: new EdgeInsets.all(20.0),
+            child: Center(
+              child: new Form(
+                key: _key,
+                child: _getFormUI(_logInModel),
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _getFormUI(LogInController _logInModel) {
+    return new Column(
+      children: <Widget>[
+        Icon(
+          Icons.cloud_circle_outlined,
+          color: Colors.lightBlue,
+          size: 150.0,
+        ),
+        new SizedBox(height: 50.0),
+        new TextFormField(
+          keyboardType: TextInputType.emailAddress,
+          autofocus: false,
+          decoration: InputDecoration(
+            hintText: 'Email',
+            contentPadding: EdgeInsets.fromLTRB(20.0, 15.0, 20.0, 15.0),
+            border:
+            OutlineInputBorder(borderRadius: BorderRadius.circular(32.0)),
+          ),
+          //validator: FormValidator().validateEmail,
+          onSaved: (String? value) {
+            _loginData.email = value!;
+          },
+        ),
+        new SizedBox(height: 20.0),
+        new TextFormField(
+            autofocus: false,
+            obscureText: _obscureText,
+            keyboardType: TextInputType.text,
+            decoration: InputDecoration(
+              hintText: 'Password',
+              contentPadding: EdgeInsets.fromLTRB(20.0, 10.0, 20.0, 10.0),
+              border:
+              OutlineInputBorder(borderRadius: BorderRadius.circular(32.0)),
+              suffixIcon: GestureDetector(
+                onTap: () {
+                  _obscureText = !_obscureText;
+                },
+                child: Icon(
+                  _obscureText ? Icons.visibility : Icons.visibility_off,
+                  semanticLabel:
+                  _obscureText ? 'show password' : 'hide password',
+                ),
+              ),
+            ),
+            //validator: FormValidator().validatePassword,
+            onSaved: (String? value) {
+              _loginData.password = value!;
+            }),
+        new SizedBox(height: 15.0),
+        new Padding(
+          padding: EdgeInsets.symmetric(vertical: 16.0),
+          child: ElevatedButton(
+            child: Text('Log In', style: TextStyle(color: Colors.white)),
+            style: TextButton.styleFrom(
```

Comment:
```suggestion
            style: ButtonStyle(
```
TextButton.styleFrom 보다는 ButtonStyle 쓰는 게 더 낫지 않을까요??! 👀

Reply:
로그인 버튼의 활성화 기능을 넣을지 말지 고민했는데 ( TEXT FIELD가 다 채워져 있는지 혹은 VALIDATION 여부에 따라 회색-> 파란색 등의 여러 기능을 고려했으나) 로그인 버튼의 경우에는 이런 기능이 필요하지 않을 것 같아서 좀 더 간편한 TextButton.styleFrom기능을 사용했습니다!